packages/forma-36-react-components/src/components/Pill/Pill.tsx

```diff
@@ -14,6 +14,8 @@ export interface PillProps {
   style?: React.CSSProperties;
   dragHandleComponent?: React.ReactNode;
   variant?: 'idle' | 'active' | 'deleted';
+  // All other props
+  [key:string]: any;
```

Comment:
Thanks for spotting this, I think a better way for TS autocomplete would be to extend the interface to include html attributes, e.g.
```ts
export interface PillProps extends HTMLAttributes<HTMLElement> {}
```

Reply:
Agreed that would be better

Reply:
Thanks